apps/src/imageUtils.js

```diff
@@ -102,7 +101,9 @@ export function imageFromURI(uri) {
 export function svgToDataURI(svg, imageType) {
   imageType = imageType || 'image/png';
   return new Promise(resolve => {
-    svg.toDataURL(imageType, {callback: resolve});
+    import('./util/svgelement-polyfill').then(() => {
+      svg.toDataURL(imageType, {callback: resolve});
+    });
```

Comment:
What does error handling look like if this import fails?

Reply:
Good question! This will trigger an error, and the component will fail to load. The docs recommend using an error boundary so that everything outside of the lazy-loaded component will continue to work: https://www.smooth-code.com/open-source/loadable-components/docs/error-boundaries/ I will see if I can add a wrapper to do this for us.

Reply:
it is also possible to add a loading spinner while the network request is happening, which would be another good default to have: https://www.smooth-code.com/open-source/loadable-components/docs/fallback/

Reply:
How does this look for default behavior?

loading:
![spin](https://user-images.githubusercontent.com/8001765/68276180-bd6a4300-0021-11ea-93d9-71be409e5716.gif)

success:
<img width="322" alt="Screen Shot 2019-11-05 at 11 05 13 PM" src="https://user-images.githubusercontent.com/8001765/68276364-d83cb780-0021-11ea-8261-99ff1425dcb3.png">

error:
<img width="349" alt="Screen Shot 2019-11-05 at 11 14 02 PM" src="https://user-images.githubusercontent.com/8001765/68276496-fbfffd80-0021-11ea-8343-e86d3843d751.png">

error, on hover:
<img width="367" alt="Screen Shot 2019-11-05 at 11 17 15 PM" src="https://user-images.githubusercontent.com/8001765/68276728-76308200-0022-11ea-8a11-bd827a227641.png">

Reply:
![yourschool](https://user-images.githubusercontent.com/8001765/68277407-f7d4df80-0023-11ea-91f5-08d2bd3224af.gif)

<img width="594" alt="Screen Shot 2019-11-05 at 11 25 13 PM" src="https://user-images.githubusercontent.com/8001765/68277412-fc999380-0023-11ea-8821-623ca8324416.png">